makefiles/pseudomodules.inc.mk

```diff
@@ -79,8 +79,10 @@ PSEUDOMODULES += sock_tcp
 PSEUDOMODULES += sock_udp
 PSEUDOMODULES += stdin
 PSEUDOMODULES += stdio_ethos
+PSEUDOMODULES += stdio_cdc_acm
 PSEUDOMODULES += stdio_uart_rx
 PSEUDOMODULES += sock_dtls
+PSEUDOMODULES += usbus_cdc_acm_uart
```

Comment:
Apparently this is never used. What was it for ?

Reply:
This should be part of #12268 and not part of this PR, thanks for spotting!